src/App.js

```diff
@@ -0,0 +1,47 @@
+import "./App.css";
+import { Message } from "./components/Message/index.js";
+import React, { useState, useEffect } from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+function App() {
+  const [MessageList, setMessageList] = useState([]);
+  const [value, setValue] = useState("");
+  const [names,setNames] = useState(['Alex','Peter','John','Alan','Phil','Bryan','Boris']);
+  const rand = Math.floor(Math.random() * names.length);
+
+  useEffect (() => {
+    console.log("App did mount");
+    }, []);
+
+  useEffect (() =>{
+    if (MessageList[MessageList.length-1]?.author === 'Boris'){
+      const timer = setTimeout(setMessageList(prevMessageList => [...prevMessageList,{text:<h4 style={{color: "red"}}>message from Bot!</h4>, author:'Bot in fact', id:`${Math.round(Math.random()*100)}`}]),1500);
```

Comment:
const timer = setTimeout(() =>{
      setMessageList(prevMessageList => [...prevMessageList,{text:<h4 style={{color: "red"}}>message from Bot!</h4>, author:'Bot in fact', id:`${Math.round(Math.random()*100)}`}]);
      },1500);

Reply:
this is correct code